test/lib/fixtures/projects/publish.js

```diff
@@ -0,0 +1,31 @@
+var retrieveTestProjects = require('./test-projects');
+
+var published = {};
+var userToken = {
+  authorization: 'token ag-dubs'
```

Comment:
I take that back! I can file a follow-up bug for this but I tried implementing this and it requires a ton of code changes because we rely on the seed data we use for regular usage of publish.